splunk/package.yaml

```diff
@@ -44,12 +38,9 @@ pipelines:
     name: Send data to Splunk
     description: Send data to a Splunk instance.
     definition: |
-      subscribe {{ inputs.events_topic }}
-      | fluent-bit
-          splunk
-          host={{ inputs.splunk_host }}
-          port={{ inputs.splunk_port }}
-          tls=on
-          tls.verify=off
-          splunk_token={{ inputs.splunk_token }}
-          event_index={{ inputs.event_index }}
+      // tql2
+      subscribe "{{ inputs.events_topic }}"
+      to_splunk "{{ inputs.splunk_url }}",
+        hec_token="{{ inputs.splunk_token }}",
+        index="{{ inputs.event_index }}",
+        skip_peer_verification = true
```

Comment:
`to` with `splunk://`?

Reply:
I could, what's the benefit?

Reply:
Actually, there is no `splunk://` scheme. So resolving this.